src/editor-sidebar/create-panel.js

```diff
@@ -32,30 +32,24 @@ import ScreenHeader from './screen-header';
 export const CreateThemePanel = ( { createType } ) => {
 	const { createErrorNotice } = useDispatch( noticesStore );
 
+	const subfolder = useSelect( ( select ) => {
+		const stylesheet = select( 'core' ).getCurrentTheme().stylesheet;
+		if ( stylesheet.lastIndexOf( '/' ) > 1 ) {
+			return stylesheet.substring( 0, stylesheet.lastIndexOf( '/' ) );
+		}
+		return '';
+	}, [] );
+
```

Comment:
I've never tried setting initial state based on a value provided by a function called within the same component, but it doesn't seem to be a problem... so I'm inclined to approve this, albeit hesitantly